Parser/string_parser.c

```diff
@@ -598,11 +370,17 @@ fstring_compile_expr(Parser *p, const char *expr_start, const char *expr_end,
         return NULL;
     }
 
-    str[0] = '(';
+    str[0] = '{';
```

Comment:
Why the change to `{`? If I remember correctly, that parenthesis were to force parsing as expressions some constructs. Is this being preserverd?

Reply:
> I think I understand why you need them when you go into `fstring_find_expr_location`

This is due to the `strstr` call in https://github.com/python/cpython/blob/c96d00e88ead8f99bb6aa1357928ac4545d9287c/Parser/string_parser.c#L522. 

> this is very subtle and certainly deserves a comment I think :)

Good point! I'll add a comment.